code/modules/modular_computers/file_system/programs/maintenance/themes.dm

```diff
@@ -0,0 +1,44 @@
+/datum/computer_file/program/maintenance/theme
+	filename = "theme"
+	filedesc = "Theme holder"
+	extended_desc = "Holds a theme you can add to your Modular PC to set in the Themify application. Makes the application use more space"
+	size = 2
+
+	///The type of theme we have
+	var/theme_name
+
+/datum/computer_file/program/maintenance/theme/New()
+	. = ..()
+	filename = "[theme_name] NtOS theme"
```

Comment:
```suggestion
	filename = "[theme_name] NtOS Theme"
```

Reply:
I think we should remove NtOS actually otherwise we get ``NtOS Dark Mode NtOS Theme``

Reply:
you do that then